cowprotocol/solver_dashboard/number_capped_reward_batches_3670624.sql

```diff
@@ -0,0 +1,33 @@
+-- This query finds the number of transactions where the rewards reached the upper bound
```

Comment:
This looks like more of an internal monitoring/special-purpose stat. Where is this query referenced?